peachjam/settings.py

```diff
@@ -83,6 +84,85 @@
 
 ROOT_URLCONF = "peachjam.urls"
 
+JAZZMIN_SETTINGS = {
+    "site_title": "Peachjam",
+    "site_header": "Peachjam",
+    "site_brand": None,
+    "site_icon": None,
+    "site_logo": "/images/africanlii-logo.svg",
```

Comment:
I think it's too awkward to make a logo work across all the sites, and this can just be unset.